pkg/api/controllers/account_controller.go

```diff
@@ -19,6 +20,29 @@ func NewAccountController() AccountController {
 	return AccountController{}
 }
 
+type BalanceOperator string
```

Comment:
Maybe move on storage package and use on file sqlstorage/account.go to avoid copy/paste?